server/services/offenderService.js

```diff
@@ -50,6 +50,9 @@ module.exports = function createOffendersService(elite2ClientBuilder) {
   }
 
   const getLocation = (token, locationId) => {
+    if (!locationId) {
```

Comment:
the main issue was this - it was always attempting to load a location - even if one hadn't been saved previously.